package.json

```diff
@@ -43,6 +44,7 @@
   },
   "dependencies": {
     "bundle-loader": "0.5.6",
+    "cheerio": "^1.0.0-rc.2",
```

Comment:
Is there some reason to import `cheerio` as a dependency or can it be imported as a devDependency instead?

Reply:
It should have been. My mistake.